lib/core/src/getRodsEnv.cpp

```diff
@@ -139,12 +142,17 @@ extern "C" {
         createRodsEnvDefaults( &rodsEnvArg );
     }
 
-    static
-    int capture_string_property(
-        const std::string&             _key,
-        char*                          _val ) {
+    static int capture_string_property(const std::string& _key, char* _val, size_t _val_size)
```

Comment:
`size_t` -> `std::size_t`

Don't forget to include the appropriate header if not included already.

I think it is defined in `<cstdint>`. Check cppreference.com to be sure.

Reply:
`std::size_t` is also defined in `<cstring>`.